modules/vstudio/_preload.lua

```diff
@@ -195,6 +178,62 @@
 		}
 	}
 
+	api.register {
+		name = "uacexecutionlevel",
+		scope = "config",
+		kind = "string",
+		allowed = {
+			"Default",
+			"AsInvoker",
+			"HighestAvailable",
+			"RequireAdministrator",
+			"Disabled"
+		},
+		aliases = {
+			Invoker = 'AsInvoker',
+			Highest = 'HighestAvailable',
+			Admin = 'RequireAdministrator',
+			None = 'Disabled'
+		}
+	}
+
+	api.register {
+		name = "uacuiaccess",
+		scope = "config",
+		kind = "string",
```

Comment:
```suggestion
		kind = "boolean",
```
It might simplify `aliases`

Reply:
did you try this? iirc there was a reason I did it this way as aliases didn't work with 'boolean' or there was some other problem.

Reply:
Didn't try it, was just a (bad) suggestion.